### ISSUE TITLE
Thoughts of merging into JNA

### ISSUE THREAD

**Issue by rednoah** [shannah/Java-Objective-C-Bridge#1]
I've only used this project to do same basic calls into Objective C (the Security Scoped Bookmarks NSURL methods) but it's been working extremely well for me, and getting things done went much more smooth than originally expected.

So first of all congrats on an amazingly simple yet powerful library! I get the feeling it's heavily under-appreciated...

Not sure if this ever came up, nor have I asked twall from JNA about this, but wouldn't your Objective-C-Bridge fit right in with the JNA project?

**Comment by LOG-TAG** [comment]
porting JNA to JNI will be a good thing?

**Comment by rednoah** [comment]
Adding a random note: `Java-Objective-C-Bridge` and `JNA` have overlapping packages (presumably so that `Java-Objective-C-Bridge` can access `JNA` protected methods) which is something the new Java Module System really really doesn't like.

**Comment by shannah** [comment]
Good point about the JPMS package overlap issue.  I have fixed it so that there is no longer a package overlap.

**Comment by rednoah** [comment]
➔ https://github.com/java-native-access/jna/issues/370
